server/src/main/java/io/druid/segment/realtime/plumber/RealtimePlumber.java

```diff
@@ -130,6 +132,17 @@
   private static final String COMMIT_METADATA_KEY = "%commitMetadata%";
   private static final String COMMIT_METADATA_TIMESTAMP_KEY = "%commitMetadataTimestamp%";
 
+  private static final ThreadMXBean THREAD_MX_BEAN = ManagementFactory.getThreadMXBean();
+
+  private static long safeGetThreadCpuTime()
```

Comment:
the other place that does something similar is CPUTimeMetricQueryRunner.safeBuild()
which is different than this.